app/assets/src/components/visualizations/heatmap/Heatmap.js

```diff
@@ -836,11 +896,28 @@ export default class Heatmap {
           this.options.onRowLabelClick(d.label, d3.event)
       );
 
+    rowLabelEnter
+      .append("line")
+      .attr("x1", 0)
+      .attr("x2", this.rowLabelsWidth)
+      .attr("y1", this.cell.height)
+      .attr("y2", this.cell.height)
+      .attr("class", cs.genusBorder)
+      .classed(cs.hideGenusBorder, (label, i, nodes) => {
+        const nextLabel = this.filteredRowLabels[i + 1];
+        if (nextLabel) {
+          return label.sortKey === nextLabel.sortKey;
+        } else {
+          // hide line at very bottom
+          return true;
+        }
+      });
+
     rowLabelEnter
       .append("text")
       .attr("class", cs.removeIcon)
       .text("X")
-      .attr("transform", `translate(0, ${this.cell.height / 2})`)
+      .attr("transform", `translate(2, ${this.cell.height / 2})`)
```

Comment:
I changed this to give the X some padding inside the gray background highlighting

Reply:
Can you put this as an option above?
Even if not changed often I prefer having this magical numbers defined above (a constant as an alternative)

Reply:
👍

Reply:
done